src/browser/documentation/help/deny.jsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react'
+import ManualLink from 'browser-components/ManualLink'
+import AdminOnSystemDb from './partials/admin-on-systemdb'
+const title = 'DENY'
+const subtitle = 'Deny privileges to roles'
+const category = 'security'
+const content = (
+  <>
+    <p>
+      The <code>DENY</code> command allows an administrator to deny a privilege
+      to a role in order to prevent access to an entity.
+    </p>
+    <div className="links">
+      <div className="link">
+        <p className="title">Reference</p>
+        <p className="content">
+          <ManualLink
+            chapter="cypher-manual"
+            page="/administration/security/subgraph/#administration-security-subgraph-introduction"
+            minVersion="4.0.0"
+          >
+            Subgraph security
+          </ManualLink>{' '}
+          manual page
+          <br />
+          <ManualLink
+            chapter="cypher-manual"
+            page="/administration/security/administration/#administration-security-administration-database-privileges"
+            minVersion="4.0.0"
+          >
+            Database administration
```

Comment:
I assume there isnt a specific `DENY` section?

Reply:
I see there are a few others like this too

Reply:
That's correct. `GRANT`, `DENY`, and `REVOKE` are together in two different sections. One for DB administration (limiting access to specific databases), and the other one for subgraph administration (limiting what kind of queries a role can do).